examples/README.md

```diff
@@ -239,16 +239,18 @@ Example | File | Description
 
 ## Shaders
 
+These examples demonstrate how to implement different shaders in user code.
```

Comment:
This is a good place to define what a "shader" is for beginners.

Reply:
I don't think shaders can be easily described in a short enough way to make sense here.

Reply:
A shader in its most common usage is a small program that is run by the GPU per-vertex in a mesh (a vertex shader) or per-affected-screen-fragment (a fragment shader.) The GPU executes these programs in a highly parallel way. There are also compute shaders which are used for more general processing leveraging the GPU’s parallelism.